tox.ini

```diff
@@ -31,17 +30,4 @@ commands =
     sphinx-build -W -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
     sphinx-build -W -b linkcheck -d {envtmpdir}/doctrees . {envtmpdir}/linkcheck
 
-# [testenv:py36]
-# basepython = python3.6
-# deps = {[testenv]deps}
-
-# [testenv:py35]
-# basepython = python3.5
-# deps = {[testenv]deps}
-
-
-# [testenv:py27]
-# basepython = python2.7
-# deps =
-#     {[testenv]deps}
-#     {[testenv]deps_py2}
+# TODO [testenv:distribute] <- streamline bulding pypi releases
```

Comment:
so you really want to ditch python 2.7 / 3.x testing? Why?
In my opinion it's quite important to make sure that the test suite completes on all target version of python